apps/angular/signal-input/src/app/user.component.ts

```diff
@@ -2,8 +2,8 @@ import { TitleCasePipe } from '@angular/common';
 import {
   ChangeDetectionStrategy,
   Component,
-  Input,
-  OnChanges,
+  computed,
+  input,
 } from '@angular/core';
 
 type Category = 'Youth' | 'Junior' | 'Open' | 'Senior';
```

Comment:
Just a little thing, but i would move that `ageToCategory` method inside the Category class as a static method, since it returns a Category object. That could be a little bit cleaner, i always try to avoid stuff outside the component class, expect for static values.

Reply:
Even for big projects, I don't use that OOP pattern. I prefer having functions than classes. 
I hate the new keyword. I prefer having to call a pure function.